### PR TITLE
Making the top of the homepage pretty

### PR DESCRIPTION
**Pull request by Darcy382**
https://user-images.githubusercontent.com/30302496/159815873-a52b30cb-5aca-4e86-b997-a946b36b21bc.mov

homepage

### CODE REVIEW

File: lib/screens/home_page.dart [hunk 1]
```diff
@@ -13,69 +15,130 @@ import '../widgets/button.dart';
...
-              return const Center(
-                child: CircularProgressIndicator(),
-              );
-            }
+                  // while loading show the progress indicator
+                  // if (events == null || registrations == null) {
+                  //   return const Center(
+                  //     child: CircularProgressIndicator(),
+                  //   );
+                  // }
```
Comment (Darcy382) [inline lib/screens/home_page.dart:76]: Commented out for testing while API is down

File: lib/screens/home_page.dart [hunk 2]
```diff
@@ -13,69 +15,130 @@ import '../widgets/button.dart';
 import '../widgets/default_text.dart';
 import '../widgets/screen.dart';
 
-class HomePage extends StatelessWidget {
+class HomePage extends StatefulWidget {
```
Comment (Darcy382) [inline lib/screens/home_page.dart:18]: Converted to stateful widget to allow for logo animation at the top of screen.

File: lib/screens/home_page.dart [hunk 3]
```diff
@@ -13,69 +15,161 @@ import '../widgets/button.dart';
...
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
+                                  Text("THIS IS A SAMPLE EVENT"),
```
Comment (Darcy382) [inline lib/screens/home_page.dart:132]: For testing scrolling on the animation until API is back.

File: lib/card_items/homepage_header.dart [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.end,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  CountdownTimerCard(),
+                  Row(
+                    children: [
+                      ElevatedButton(
+                        onPressed: () async {
+                          await launch("https://www.youtube.com/watch?v=dQw4w9WgXcQ");
```
Comment (rsquared226) [inline lib/card_items/homepage_header.dart:48]: wtf is this

File: lib/card_items/homepage_header.dart [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                            Icon(Icons.favorite, color: ThemeColors.StadiumOrange,),
+                            Container(width: 10,),
+                            DefaultText("SUBMIT", textLevel: TextLevel.button, color: ThemeColors.StadiumOrange,),
+                          ],
+                        ),
+                      ),
+                      Container(width: 20,),
+                      ElevatedButton(
+                        onPressed: () async {
+                          await launch("https://www.youtube.com/watch?v=dQw4w9WgXcQ");
```
Comment (rsquared226) [inline lib/card_items/homepage_header.dart:65]: stop

**Review by rsquared226 — APPROVED** [review]
sexiest thing ive seen

**Pull request merged**